x-pack/plugins/infra/common/log_sources/resolved_log_source_configuration.ts

```diff
@@ -91,8 +92,8 @@ const resolveKibanaIndexPatternReference = async (
 
   return {
     indices: indexPattern.title,
-    timestampField: indexPattern.timeFieldName ?? '@timestamp',
-    tiebreakerField: '_doc',
+    timestampField: TIMESTAMP_FIELD,
```

Comment:
Thanks for making these changes based on the feedback from the other PR. 

I *think* for the data view / KIP version we still want to read from the data view itself first. 

`timestampField: indexPattern.timeFieldName ?? TIMESTAMP_FIELD`

(@weltenwort Would you say this is right, to honour the data view setting first? Or do we always want to go with the ECS constant?)

Reply:
Yes, I'd agree that we still want to use the data view's `timeFieldName` in that case. :+1: